eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -764,7 +764,7 @@ case class Commitments(params: ChannelParams,
 
   // @formatter:off
   // HTLCs and pending changes are the same for all active commitments, so we don't need to loop through all of them.
-  def isIdle: Boolean = active.head.isIdle(changes)
+  def isIdle: Boolean = active.head.isIdle(changes, params.remoteParams.initFeatures.hasFeature(QuiescePrototype))
```

Comment:
Nice, good idea to use the feature bits instead of exposing the boolean. I think we should check both local and remote features though, we apply quiescence only if both nodes advertise it.

Reply:
Fixed in [9fa905a](https://github.com/ACINQ/eclair/pull/2680/commits/9fa905a3e557420b011b8c5079cb48ba74b705a2)